### DISCUSSION TITLE
Original Dexcom Reciver loses connection.

### DISCUSSION THREAD

**Discussion by JayJay265** [NightscoutFoundation/xDrip#3621] [Bug reports]
I use the original Dexcom receiver and a Samsung S10 with Android 11. Today I worked a little through the Xdrip versions. I noticed that everything works correctly up to version 210203. However, as soon as I have install newer version from 210204 -> 210521 the Dexcom receiver cannot establish a connection to the transmitter. After I install the Version 210203 at the next Connection Xdrip and original Reciver Shows the new value. 
Firmware of the Dexcom Receiver 5.1.2.026

![Screenshot_20210402-215734_xDrip+](https://user-images.githubusercontent.com/80785292/113449597-78a53000-93fe-11eb-8072-0b4ba5db41a9.jpg)

**Comment by englishmastiff2** [comment]
I have a Bluetooth issue after the update on April 6th. My phone's Bluetooth shuts down every 5mins readings on xdrip no longer work. Only my Dexcom reader works. Super annoying it was fine before the update on my last sensor and transmitter. I changed the transmitter, paired it, did the warm-up with both devices. The phone with xdrip and Dexcom reader. I have no readings on phone anymore and Dexcom reader works fine. Is anyone else having this issue?

**Comment by Navid200** [comment]
@englishmastiff2 Why have you posted in this thread?
Would you please either post in the facebook group or the Gitter room?
https://www.facebook.com/groups/xDripG5
https://gitter.im/jamorham/xDrip-plus

**Comment by englishmastiff2** [comment]
Unfortunately, I tried to join the Facebook group but they did not accept
my request. I don't know where else to ask I also posted on Reddit. The app
on my phone is not working.

On Wed, Apr 7, 2021 at 2:41 PM Navid200 ***@***.***> wrote:

> @englishmastiff2 <https://github.com/englishmastiff2> Why have you posted
> in this thread?
> Would you please either post in the facebook group or the Gitter room?
> https://www.facebook.com/groups/xDripG5
> https://gitter.im/jamorham/xDrip-plus
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/NightscoutFoundation/xDrip/issues/1692#issuecomment-815247876>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATSOCN36ODG6KKWBCO7RW53THS7OHANCNFSM42JJ7LMA>
> .
>


-- 
*Aaron Tange*
www.myparcourse.com

*The World is my Gym*

**Comment by Navid200** [comment]
@englishmastiff2 Please post in the Gitter room:
https://gitter.im/jamorham/xDrip-plus

**Comment by englishmastiff2** [comment]
Will do, thanks!

On Wed, Apr 7, 2021 at 2:46 PM Navid200 ***@***.***> wrote:

> @englishmastiff2 <https://github.com/englishmastiff2> Please post in the
> Gitter room:
> https://gitter.im/jamorham/xDrip-plus
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/NightscoutFoundation/xDrip/issues/1692#issuecomment-815251183>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATSOCNYLDWQNOWFAPFS25C3THTABTANCNFSM42JJ7LMA>
> .
>


-- 
*Aaron Tange*
www.myparcourse.com

*The World is my Gym*

**Comment by Navid200** [comment]
@JayJay265 Would you mind performing an experiment?
Please install a recent xDrip release that disables the receiver.
After confirming that the receiver is not receiving readings, please go to the following:

Settings -> Hardware Data Source

Scroll all the way down to the bottom and tap on "Disable Collection".  Now xDrip will not show readings anymore.  

Does the receiver work now?

To go back, just go back to Hardware Data Source and tap on G5/G6 Transmitter to restore xDrip operation.

**Comment by JayJay265** [comment]
@Navid200 I've tried it now. Version 210406 ​​installed. Receiver no longer receives any data. Xdrip set to "Dialsable Collector" Receiver can now receive data again. Xdrip setting reset Reciver does not receive any data. I repeated this three times, each time the same result.

**Comment by Navid200** [comment]
@JayJay265 Thanks
Would you mind posting a screenshot of the following page?  You can take a longshot to keep it in a single image.
Settings -> G5/G6 Debug Settings

Thanks

**Comment by JayJay265** [comment]
Version 210203
![Screenshot_20210408-003242_xDrip+](https://user-images.githubusercontent.com/80785292/113943276-9c9fb180-9802-11eb-8ba4-3a6788bf9f5b.jpg)

**Comment by Navid200** [comment]
5 days ago, Voltage B was 257.
What is Voltage B now?

**Comment by JayJay265** [comment]
I have change the Transmitter for 2 days 

<details >
<summary>Screenshot Transmitter Information</summary>
<br>
<img src="https://user-images.githubusercontent.com/80785292/113981432-f5456d80-9847-11eb-9303-4672f7129197.jpg" width="500">
</details>

**Comment by Navid200** [comment]
@JayJay265 And are you experiencing the same problem with this transmitter?
Why are the battery voltages so low?  Are they rechargeable?

**Comment by JayJay265** [comment]
It is the same with this transmitter. No it is not rechargeable i replace the batterie. Maybe the batterie has no high quality or because i replaced the batterie in february

**Comment by Navid200** [comment]
I posted in the facebook group to see if anyone else may be experiencing the same issue.
https://www.facebook.com/groups/xDripG5/permalink/5238310539575732/

No one so far.

**Comment by englishmastiff2** [comment]
Xdrip started getting readings again last night after reverting to earlier
version and changing to recommended settings.

**Comment by Navid200** [comment]
@englishmastiff2 You are receiving emails, every time someone posts, because you have automatically subscribed to this issue by posting in it.    You can unsubscribe.
This is an open issue and there will be lots of posts.  We need to communicate with the person who has asked for help.
Would you please stop responding to every post?

**Comment by Navid200** [comment]
@JayJay265 It is possible the the receiver is not as sensitive as a phone.  I suspect that since the transmitter battery voltage is so low, the transmitted signal is not strong enough for the receiver.
But, this does not explain why the receiver's behavior changes when you use a different xDrip release!

I will be very interested in the behavior with a better transmitter battery.  For example, a Voltage B of 290.  Or, at least, 285.

**Comment by JayJay265** [comment]
So the lowest voltage with which I have ever seen was almost 240 but then it was also over with both devices. On the weekend i change the Transmitter for testing.

**Comment by JayJay265** [comment]
@Navid200 So today I used a different transmitter Voltage B 289. When starting the sensor, the Dexom receiver ended the warm-up time five minutes before Xdrip. After the first values ​​also arrived at Xdrip, I installed the version from April.

Since there was now a loss of signal in the receiving values, even though all devices were all in the same positions; I create a small table.

<details >
<summary>table</summary>
<br>
<table class="tg">
<thead>
  <tr>
    <th class="tg-c3ow">value</th>
    <th class="tg-c3ow">Dexcom</th>
    <th class="tg-c3ow">Xdrip</th>
    <th class="tg-c3ow">After lost Connection Dexcom</th>
    <th class="tg-c3ow"></th>
  </tr>
</thead>
<tbody>
  <tr>
    <td class="tg-c3ow">1</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow">no,xdrip is <br>still warming<br>up</td>
    <td class="tg-c3ow"></td>
    <td class="tg-c3ow">Xdrip submitted with<br>second or third value</td>
  </tr>
  <tr>
    <td class="tg-c3ow">2</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow">yes<br>version 210302</td>
    <td class="tg-c3ow"></td>
    <td class="tg-c3ow"></td>
  </tr>
  <tr>
    <td class="tg-c3ow">3</td>
    <td class="tg-c3ow">no</td>
    <td class="tg-c3ow">yes<br>version 210604</td>
    <td class="tg-c3ow"></td>
    <td class="tg-c3ow">Dexcom backfilled <br>at some point</td>
  </tr>
  <tr>
    <td class="tg-c3ow">4</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow">no Signal lost</td>
    <td class="tg-c3ow">Backfilled</td>
    <td class="tg-c3ow"></td>
  </tr>
  <tr>
    <td class="tg-c3ow">5</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow">no Signal lost</td>
    <td class="tg-c3ow">Backfilled</td>
    <td class="tg-c3ow"></td>
  </tr>
  <tr>
    <td class="tg-c3ow">6</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow"></td>
    <td class="tg-c3ow"></td>
  </tr>
  <tr>
    <td class="tg-c3ow">7</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow">no Signal lost</td>
    <td class="tg-c3ow">Backfilled</td>
    <td class="tg-c3ow"></td>
  </tr>
  <tr>
    <td class="tg-c3ow">8</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow">no Signal lost</td>
    <td class="tg-c3ow">Backfilled</td>
    <td class="tg-c3ow"></td>
  </tr>
  <tr>
    <td class="tg-c3ow">9</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow"></td>
    <td class="tg-c3ow"></td>
  </tr>
  <tr>
    <td class="tg-c3ow">10</td>
    <td class="tg-c3ow">Signal <br>lost</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow"></td>
    <td class="tg-c3ow">Dexcom backfilled<br>with 22 value</td>
  </tr>
  <tr>
    <td class="tg-c3ow">11-20</td>
    <td class="tg-c3ow">Signal<br>lost</td>
    <td class="tg-c3ow">yes</td>
    <td class="tg-c3ow"></td>
    <td class="tg-c3ow">Dexom backfilled <br>with 22 value</td>
  </tr>
  <tr>
    <td class="tg-0pky">21</td>
    <td class="tg-0pky">Signal<br>lost</td>
    <td class="tg-0pky"> yes<br>install versio<br>210203</td>
    <td class="tg-0pky"></td>
    <td class="tg-0pky"></td>
  </tr>
  <tr>
    <td class="tg-0pky">22</td>
    <td class="tg-0pky">yes </td>
    <td class="tg-0pky">yes </td>
    <td class="tg-0pky"></td>
    <td class="tg-0pky"></td>
  </tr>
</tbody>
</table>
</details>

<details >
<summary>Screenshot log</summary>
<br>
<img src="https://user-images.githubusercontent.com/80785292/114271351-36d54480-9a11-11eb-8edc-10e2d03e0a1d.jpg" width="500">


</details>
<details >
<summary>Screenshot Transmitter Information</summary>
<br>
<img src="https://user-images.githubusercontent.com/80785292/114270660-eb6d6700-9a0d-11eb-9cb0-3d2d7786998d.jpg" width="500">
</details>

**Comment by Navid200** [comment]
I was hoping for a summary.
You changed the transmitter to one with a better battery (Voltage B of 289).
1- The problem is resolved.
2- The problem remains.

Which one is it?

**Comment by JayJay265** [comment]
What Do you mean with summary?
 2 The problem remains.

**Comment by Navid200** [comment]
@JayJay265 What is the status now?
Are all your settings matching the following?
https://github.com/Navid200/xDrip/wiki/G6-Recommended-Settings

**Comment by JayJay265** [comment]
@Navid200 The settings are the same as on the website. Now it has happened twice that the Dexcom Reciver receives data  but Xdrip not I will watch it a little bit longer

**Comment by Navid200** [comment]
@JayJay265 Is Voltage B still low?

**Comment by JayJay265** [comment]
Voltage B 260 is up to date. After I updated Xdrip to 06.11 this morning, either the Dexcom receiver got values ​​or Xdrip never got both at the same time. One of them was signal loss. After about 2 hours there are no more values ​​on the Dexcom Reciver only with Xdrip. I could change the transmitter at the weekend but don't know if I have batteries that come with Voltage B 280.

**Comment by JayJay265** [comment]
@Navid200 
I have now used a different transmitter voltage B317. However, the problem has not changed.
![Screenshot_20210630-052933_xDrip+](https://user-images.githubusercontent.com/80785292/123952232-ff3dbf00-d9a5-11eb-9ab1-9e92f76e0450.jpg)

**Comment by Navid200** [comment]
@JayJay265
That can't be right.  Voltage A cannot be 220!
Something is very wrong with those values (vA < vB, R too large).

**Comment by Navid200** [comment]
@JayJay265 
Are you still experiencing this problem?
If yes, would you please confirm, again, that the cross over from working to not working is 2021.02.03 to 2021.02.04?
Would you also confirm that the problem persists with the latest Nightly, 2021.08.04?

**Comment by JayJay265** [comment]
@Navid200 
Yes the problem is still there. Only the version from 2021.02.03 works, all afterwards have the same error, including the latest from 2021.08.04.

**Comment by Navid200** [comment]
@JayJay265 
Something changed, in the xDrip code, from Feb. 3 to Feb. 4 that is causing this considering that the same transmitter works fine with the same receiver with one version and not with the other.
Most changes to the Dexcom collector are done to improve functionality for the majority.
This behavior you are experiencing is not the intended behavior the changes were made for.

I can run experiments hoping to find exactly which part of the new code is the reason, change the xDrip code to keep the expected xDrip functionality but remove this unintended behavior, and submit it for release to the public.

The problem I can anticipate is to convince the developers to approve it.  As soon as I submit this, the question will come up "why is the transmitter battery voltage so low".  I can explain to them everything I have stated in the first paragraph above and say that the battery is irrelevant.  If it works with the same receiver and with one version of xDrip, why shouldn't it work with another version of xDrip.
But, I will not have an answer to the question, "why is the battery voltage so low".

If the submission is never approved to be released, I will not have solved any problems.

Is there any chance you can get your hands on a transmitter with expected battery voltages and resistance?

**Comment by JayJay265** [comment]
@Navid200 
![Screenshot_20210807-170435_xDrip+](https://user-images.githubusercontent.com/80785292/128604615-e321e020-b6b6-4163-b6ae-54e7f4547d9e.jpg)
Found a Transmitter with Voltage 290. But nothing Change.

**Comment by Navid200** [comment]
@JayJay265 So, with this transmitter, the receiver works fine with 2021.02.03 but the receiver stops working with 2021.02.04, while xDrip works fine either way, right?

Would you please also post a screenshot of the following page?
Settings -> G5/G6 Debug Settings

**Comment by JayJay265** [comment]
@Navid200 Yes the Reciver work with 2021.02.03 and stop with 2021.02.04. XDrip work with all Versions. 
![Screenshot_20210807-172159_xDrip+](https://user-images.githubusercontent.com/80785292/128605191-dfbb6249-f7c9-4f21-9f2d-9451601913c3.jpg)
